internal/app/swaggerui/swaggerui.go

```diff
@@ -55,7 +56,7 @@ func serve(cfg config.View) {
 	mux := &http.ServeMux{}
 	closer := telemetry.Setup(mux, cfg)
 	defer closer()
-	port := cfg.GetInt("api.swaggerui.httpport")
+	port := cfg.GetInt(internal.SwaggerUIPrefix + internal.HTTPPortSuffix)
```

Comment:
Since we are using consts anyways, trying to append these seems odd. We might as well have each of these as first class constants? (SwaggerUIHTTPPort.. ). I understand there ll be many then - but they are scoped to a consts file and the rest of the usage is clean.